cleansweep/plugins/committees/views.py

```diff
@@ -175,6 +175,14 @@ def committee_structures():
     place = Place.get_toplevel_place()
     return render_template("committee_structures.html", place=place)
 
+
+@plugin.route("/admin/committee-structures/at/<level>")
+@require_permission("admin.committee-structures.view")
+def committees_by_level(level):
+    place_type = PlaceType.get(level)
+    place = Place.get_toplevel_place()
+    return render_template("view_committees_by_level.html", place_type=place_type, place=place)
```

Comment:
Umm yeah. Since we have both committee and committee structure files in same folder. I'll change it.